NumLib/Fem/ShapeMatrixPolicy.h

```diff
@@ -34,7 +34,8 @@ struct EigenFixedShapeMatrixPolicy
         NumLib::ShapeMatrices<
             NodalVectorType,
             DimNodalMatrixType,
-            DimMatrixType>;
+            DimMatrixType,
+            DimNodalMatrixType>;
```

Comment:
indeed this is a bug. I should pass a matrix type having a size of global dimension x number of nodes. see the latest where the bug is fixed.